ONSdigital/eq-questionnaire-runner
# Update previous behaviour after coming from summary page

### What is the context of this PR?

Updates the behaviour so that if you go back to a question page from the summary page, when you click the `previous` button, then you return back to the summary. 

### How to review 

Use the `test_section_summary` or `test_submit_with_summary` schema and
 - Complete the survey
 - Use the `change` link on the `Summary` page to edit a question
 - On the `Question` page hit `previous`
 - Check that you are returned to the Summary page

### Checklist
